static/docs/commands-reference/status.md

```diff
@@ -55,15 +56,32 @@ DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
 differ are listed. For each item listed, either the file name or the checksum is
 shown, and additionally a status word is shown describing the change:
 
-- For the local workspace:
-  - _changed_ means the named file has changed
+- For the changes detected in local workspace:
+
+  - _changed deps_ means the DVC-file has changed dependencies
+  - _changed outs_ means the DVC-file has changed outputs
+  - _changed checksum_ means actual checksum of the DVC-file doesn't match the
+    one specified in it in DVC-files
+  - _always changed_ means that this is a special DVC-file with no dependencies,
+    which is considered always changed
+
+- For the output displayed in local workspace:
```

Comment:
is it the same for dependencies?

Reply:
Yes, it is the same for dependencies too.